ivy/functional/backends/torch/activations.py

```diff
@@ -71,3 +71,15 @@ def softplus(x: torch.Tensor, /) -> torch.Tensor:
 
 
 softplus.unsupported_dtypes = ("float16",)
+
+
+def mish(
```

Comment:
Is this part of the task?

Reply:
This was a mistake; I accidentally pushed to the wrong branch. I'll fix it in the next commit, and sure I'll add some examples :)
Thanks!